README.md

```diff
@@ -2,60 +2,61 @@
 
 [![Build Status](https://travis-ci.org/opensupports/opensupports.svg?branch=master)](https://travis-ci.org/opensupports/opensupports) v4.1.1
 
-OpenSupports is an open source ticket system built primarly with PHP and ReactJS.
+OpenSupports is an open source ticket system built primarily with PHP and ReactJS.
 Please, visit our website for more information: [http://www.opensupports.com/](http://www.opensupports.com/)
 
 ## Requirements
 * PHP 5.6+
 * MySQL 4.1+
 
 ## Development
-Here is a guide of how to set up the development environment in OpenSupports
+Here is a guide of how to set up the development environment in OpenSupports.
 
 ### Getting up and running FRONT-END (client folder)
-0. update `sudo apt-get update`
-1. Clone this repo
-2. Install node 4.x version
+1. Update: `sudo apt-get update`
+2. Clone this repo: `git clone https://github.com/opensupports/opensupports.git`
+3. Install node 4.x version:
     - `sudo apt-get install curl`
     - `curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -`
     - `sudo apt-get install -y nodejs`
-3. Install npm `sudo apt-get install npm`
-4. Install gulp `sudo npm install -g gulp`
-5. Go to repo `cd os4-react/client`
-6. Install dependences `npm install`
-7. Rebuild node-sass `npm rebuild node-sass`
-8. Run `gulp dev`
-9. Go to the main app: `http://localhost:3000/app` or the component demo `http://localhost:3000/demo`
-10. Your browser will automatically be opened and directed to the browser-sync proxy address
-12. Use `gulp dev --api` to disable fixtures and use the real php server api (it must be running at :8080).
+4. Install npm: `sudo apt-get install npm`
+5. Install gulp: `sudo npm install -g gulp`
+6. Go to client: `cd os4-react/client`
```

Comment:
I think we should change it to `cd opensupports/client`, same goes for line 53 and 87

Reply:
Agree. Fixed now.